.jest/setEnvVars.ts

```diff
@@ -0,0 +1,2 @@
+//SET ENV VARIABLES FOR TESTS
+process.env.TEST_DATABASE_NAME = 'open_forum_test_db';
```

Comment:
Since this is a test DB does this need to be an environment variable?

Reply:
It seems strange to have your test set-up file declare an environment variable. If you really wanted to go this route, why not just add it to your .env file?

Reply:
@JeromeGoodrich Yea, I only added this because Jest has a different environment from the current application. So it didn't read the application's .env file. I think I'll just hard-code the database name.